### ISSUE TITLE
MySQL?

### ISSUE THREAD

**Issue by mzaini30** [bkader/mini-server-codeigniter#1]
Can I use MySQL?

**Comment by bkader** [comment]
Yes, it has built-in MySQL as well. But this this kind of old unmaintained project. [See here](https://github.com/bkader/mini-server-codeigniter/blob/a8393ae20fc226e19c57315e8295d7f78121b4c4/root/core/core/CodeIgniter.php#L58).

**Comment by mzaini30** [comment]
I've try click `mysql_start.bat` then I can result `The system cannot find the file specified.`. What's wrong?
